diff --git a/dmenu.c b/dmenu.c
index 8d9bbb6..3f0e78d 100644
--- a/dmenu.c
+++ b/dmenu.c
@@ -612,6 +612,12 @@ setup(void) {
 	xic = XCreateIC(xim, XNInputStyle, XIMPreeditNothing | XIMStatusNothing,
 	                XNClientWindow, win, XNFocusWindow, win, NULL);
 
+    XClassHint *ch = XAllocClassHint();
+    ch->res_name = "dmenu";
+    ch->res_class = "dmenu";
+    XSetClassHint(dc->dpy, win, ch);
+    XFree(ch);
+
 	XMapRaised(dc->dpy, win);
 	resizedc(dc, mw, mh);
 	drawmenu();
